kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/Product.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.good.patterns.challenges;
+
+public class Product {
+
+    private String nameOfProduct;
+    private double priceOfProduct;
```

Comment:
Because it is only example, price can be represented as a double, but in a normal product you have to use BigDecimal - see this discussion: https://stackoverflow.com/questions/3413448/double-vs-bigdecimal